tests/modules/nf-core/phispy/test.yml

```diff
@@ -0,0 +1,12 @@
+- name: phispy test_phispy
+  command: nextflow run ./tests/modules/nf-core/phispy -entry test_phispy -c ./tests/config/nextflow.config -c ./tests/modules/nf-core/phispy/nextflow.config
+  tags:
+    - phispy
+  files:
+    - path: output/phispy/test/test_genome.gbff.gz
```

Comment:
Can you also remove this `_genome` suffix?

Reply:
Sure, that will just take a bit more work because I'll have to capture the file extension, since it's based on the file extension in the input file.

Reply:
Done - https://github.com/nf-core/modules/pull/3453#discussion_r1205514864